src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -449,8 +509,111 @@ final case class AgreementApiServiceImpl(
           verifiedAttributes = agreement.verifiedAttributes,
           suspendedByConsumer = suspendedByConsumer,
           suspendedByProducer = suspendedByProducer,
-          suspendedByPlatform = suspendedByPlatform
+          suspendedByPlatform = suspendedByPlatform,
+          stamps = agreement.stamps.copy(suspension = None)
```

Comment:
The intent of the stamp field is to freeze some legal moments relating to the life cycle of the agreement.
The only exception is the value of the suspension because the suspension has no contractual significance.